machine1/dbus_test.go

```diff
@@ -157,7 +157,10 @@ func generateRandomLabel(n int) string {
 
 func TestImages(t *testing.T) {
 	imageName := machinePrefix + generateRandomLabel(8)
-	imagePath := filepath.Join("/var/lib/machines", imageName)
+	imagePath := filepath.Join("/var/lib/machines", imageName+".raw")
+
+	newImageName := machinePrefix + generateRandomLabel(8)
+	newImagePath := filepath.Join("/var/lib/machines", newImageName+".raw")
```

Comment:
Is this used anywhere?

Reply:
It's used in CloneImage test.

Reply:
I'm sorry, I'm not following. `newImagePath` is used exactly once in a `os.Remove`; who is creating and populating that file?

Reply:
conn.CloneImage will copy the image to new place.